### PR TITLE
Fixed too big text issue

### LINKED ISSUE #138: Bug: Too long text in toolbar titles and buttons
When a text is too long it has no functionality for handling this. Solve so that text automatically shrinks to some extent then that it is handled in a good way.

### PR DESCRIPTION
**Pull request by aolsson711**
Made all text in app using auto size and if the size still doesn't fit "..." will come after

closes #138

### CODE REVIEW

**Review by magangloff — APPROVED** [review]
All good.

File: app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void TestContentOfListView(){
         for (int i = 0; i < 20; i++){
             DataInteraction listItem = interaction.atPosition(i);
 
-            String name = String.format(Locale.getDefault(),"TEST_Mountain%d", i);
+            String name = String.format(Locale.getDefault(),"Mont Blanc - Monte Bianco%d", i);
```
Comment (bgabriel998) [inline app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java:84]: This test will be replaced with the one in `map_activity` branch

File: app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java [hunk 2]
```diff
@@ -95,7 +95,7 @@ public void TestContentOfListView(){
     public void TestPressCollected() {
         // Item at pos 10 looks like this
         CollectedItem correctItem = new CollectedItem(
-                String.format(Locale.getDefault(),"TEST_Mountain%d", 10),
+                String.format(Locale.getDefault(),"Mont Blanc - Monte Bianco%d", 10),
```
Comment (bgabriel998) [inline app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java:98]: This test will be replaced with the one in `map_activity` branch

File: app/src/main/java/com/github/bgabriel998/softwaredevproject/CollectionActivity.java [hunk 3]
```diff
@@ -47,7 +47,7 @@ private void fillCollectionList(){
         ArrayList<CollectedItem> collectedItems = new ArrayList<>();
 
         for (int i = 0; i < 20; i++){
-            collectedItems.add(new CollectedItem(String.format(Locale.getDefault(),"TEST_Mountain%d", i), 100 - i, 1000-i, 50+i,50-i));
+            collectedItems.add(new CollectedItem(String.format(Locale.getDefault(),"Mont Blanc - Monte Bianco%d", i), 100 - i, 1000-i, 50+i,50-i));
```
Comment (bgabriel998) [inline app/src/main/java/com/github/bgabriel998/softwaredevproject/CollectionActivity.java:50]: This function will be replaced with the one in `map_activity` branch

**Review by bgabriel998 — APPROVED** [review]
Take note of the comments left on the PR for futur merge. Otherwise it is all good

File: app/src/main/java/com/github/bgabriel998/softwaredevproject/CollectionActivity.java (hunk 3 above)
Comment (bgabriel998) [reply app/src/main/java/com/github/bgabriel998/softwaredevproject/CollectionActivity.java]: Resolved with merge from master

File: app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java (hunk 2 above)
Comment (bgabriel998) [reply app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/CollectionActivityTest.java]: Resolved with merge of master

**Pull request merged**